psc-docs/Ctags.hs

```diff
@@ -1,5 +1,6 @@
 module Ctags (dumpCtags) where
 
+import Prelude
```

Comment:
So that's every module bar a few in `psc-ide` right now. I think that's too much honestly. Can't these small patches be maintained locally instead?